_events/18.md

```diff
@@ -0,0 +1,38 @@
+---
+title:  "General Meeting - #7"
+date:   2020-01-30 19:30:00 +0100
+tags:
+   -general-meeting
+---
+
+## {{ page.date | date: '%A, %d %B %Y' }}
+
+## TWC General Meeting
```

Comment:
either there should be text underneath line 8 (stuff on line 13), or we can merge line 8 and 10